### ISSUE TITLE
"The URL appears to be cut short or malformed" message

### ISSUE THREAD

**Issue by scsmash3r** [porsager/flems.io#21]
Target website: https://flems.io
Problem description: I've came with a medium-sized module with 5 files and 1 jQuery dependency. After working with Flems, I ended up with an issue: the URL, generated by Flems, cannot be copied fully from browser's URL bar (in my case URL length is 380 000+ symbols), and in Chrome browser (v.73.0.*) there is a limit of 32779 chars (string gets truncated). So that means I can not share my Flems, using correct URL, cause of browser's in-built URI input limits.

See also: https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

What can be done about this?.. Should there be a warning posted somewhere for users, so they would not lost their work in the process?

**Comment by porsager** [comment]
I'll respond more in depth when I'm at my computer, but for now you could download the Flems not to loose it. 

In the top of the left sidebar there is a download button.

**Comment by scsmash3r** [comment]
> I'll respond more in depth when I'm at my computer, but for now you could download the Flems not to loose it.
> 
> In the top of the left sidebar there is a download button.

Yes, I'm aware of download button. I also managed to copy full link from the 'History' section of browser. It then let me copy all of 381 496 symbols and, weirdly enough, I was able to paste it into URL bar and simply go to my active project.

Here is the sample link for you to try: https://gist.github.com/scsmash3r/27a1eea8cb24fcc5afc1b14f764dfb79
